Mention kernel-default-base doesn't include iscsi_tcp (#920)
longhorn/website: content/docs/1.6.2/deploy/install/_index.md

modprobe iscsi_tcp
```

> Note that on SUSE and openSUSE, the iscsi_tcp module is included in the `kernel-default` package, but
> is not available in `kernel-default-base`. If you have `kernel-default-base` installed, you will need
> to install `kernel-default` instead.

Reviewer: ```suggestion
> **Important**: On SUSE and openSUSE, the `iscsi_tcp` module is included only in the `kernel-default` package. If the `kernel-default-base` package is installed on your system, you must replace it with `kernel-default`.
```
